browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -2637,6 +2637,9 @@ L.Control.JSDialogBuilder = L.Control.extend({
 			return;
 		}
 
+		// eg. in mobile wizard input is inside spin button div
+		var innerInput = control.querySelector('input');
```

Comment:
no, please do not do it that way (like it was in the past I think).

The idea is: we have "black box" widget - we setup on its only node disabled state and it cares itself about inner structure.

We have special helper for that case: https://github.com/CollaboraOnline/online/blob/master/browser/src/control/jsdialog/Util.StateChange.ts#L35

example use: https://github.com/CollaboraOnline/online/blob/fbb6a691c77069da25a9e72921bc3cb469ab6641/browser/src/control/jsdialog/Widget.Edit.ts#L88


let me fix that

Reply:
yeah I was not happy with this PR either, specially because it ends up mixing again Mobile stuff with the non Mobile stuff